### ISSUE TITLE
Support custom trusted apps

### ISSUE THREAD

**Issue by Laurin-W** [activitypods/activitypods#312]
Support configuring different sources for trusted apps or adding custom ones to the list from the APods website.
Right now they are loaded from here only: https://github.com/activitypods/activitypods/blob/10978debfc55c95b362efca52352be282ea07daa/pod-provider/frontend/src/config/dataServers.js#L32

**Comment by srosset81** [comment]
I agree ! In the long term, the idea would be to support multiple "app stores", to avoid this aspect to be centralized, since there is no need for that. Pod providers could choose what app stores they want to list. Eventually this could be chosen by the users themselves.
